source/elements/oneDPL/source/parallel_api.rst

```diff
@@ -65,7 +65,7 @@ where and how an algorithm runs.
         template <typename KernelName = /*unspecified*/>
         class device_policy;
 
-        device_policy<> dpcpp_default;
+        inline const device_policy<> dpcpp_default{};
```

Comment:
I think the way `dpcpp_default` is constructed should not be prescribed in the specification. At most, we should say that the SYCL queue associated with `dpcpp_default` should compare equal to a queue associated with any default-constructed `device_policy`.

Reply:
ok